agent_application/example.config.yaml

```diff
@@ -24,6 +24,7 @@ signing_algorithms_supported:
     enabled: false
 
 # TODO: required to be stated explicitly?
+# TODO: update to vp_formats_supported
```

Comment:
Couldn't this TODO be resolved quickly by just providing the default config value as an example?